sigle/src/modules/analytics/stats/StatsTotal.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { Box, Flex, Text } from '../../../ui';
+import { StatsData } from './types';
+
+interface TotalViewsAndVisitorsProps {
+  pageviews: number;
+  visits: number;
+}
+
+interface StatsTotalProps {
+  data: StatsData[];
+}
+
+const numberWithCommas = (x: string): string => {
+  return new Intl.NumberFormat('en-US').format(Number(x)).toString();
+};
+
+export const StatsTotal = ({ data }: StatsTotalProps) => {
+  const [totalViewsAndVisitors, setTotalViewsAndVisitors] =
+    useState<TotalViewsAndVisitorsProps>();
+
+  useEffect(() => {
```

Comment:
I feel like this could be changed to `useMemo` so we can avoid rendering it twice :)